src/scss/06_components/paragraphs/_react-search.scss

```diff
@@ -98,3 +98,52 @@ fieldset .react-search__checkbox:not(:first-of-type) {
   }
   margin-top: $spacing-double;
 }
+
+.hdbt-search--react {
+  .react-search__form-container {
+    border-bottom: none;
+    @include breakpoint($breakpoint-l) {
+      max-width: 870px;
+    }    
+  }
+
+  .hdbt-search__submit-button {
+    width: 100%;
+
+    @include breakpoint($breakpoint-l) {      
+      padding-left: 130px;
+      padding-right: 130px;
```

Comment:
I think we should use width (max/min) to control button width instead of padding.

Reply:
Using `min-width` now.